test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -563,10 +585,10 @@ void testBCTest(json_spirit::mObject& _o)
 	ImportTest::compareStates(postState, blockchain.topBlock().state());
 }
 
-bigint calculateMiningReward(u256 const& _blNumber, u256 const& _unNumber1, u256 const& _unNumber2)
+bigint calculateMiningReward(u256 const& _blNumber, u256 const& _unNumber1, u256 const& _unNumber2, bool isByzantium)
 {
 	unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(test::TestBlockChain::s_sealEngineNetwork)).createSealEngine());
-	bigint baseReward = se->chainParams().blockReward;
+	bigint baseReward = se->chainParams().blockReward(isByzantium);
```

Comment:
ChainParams  constructed from genesisInfo so it does know all the transition blocks. 
Schedule variables are: EVMSchedule.h

Ah, and here there is a function 
EVMSchedule const& SealEngineBase::evmSchedule(u256 const& _blockNumber) const